ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/verticaltabs/v1/verticaltabs/verticaltabs.html

```diff
@@ -58,7 +58,7 @@
       data-cmp-hook-adaptiveFormVerticalTabs="tabpanel"></div>
 
 </div>
-    <sly data-sly-resource="${resource.path @ resourceType='wcm/foundation/components/parsys/newpar', appendPath='/*', decorationTagName='div', cssClassName='new section aem-Grid-newComponent'}"
+    <sly data-sly-resource="${resource.path @ resourceType='core/wcm/components/container/v1/container/new', appendPath='/*', decorationTagName='div', cssClassName='new section aem-Grid-newComponent'}"
```

Comment:
Let's discuss in scrum, if this might cause any backward incompatibility and if a new version of component is required.

Reply:
I just checked since we are re-using the same CSS selectors, I don't see any backward incompatibility, LGTM